output_template/python/run.py

```diff
@@ -148,6 +162,61 @@ def run_prediction(input_image, model, config_file, max_percent_incorrect_values
     _save_images(output_dir, filename_images)
 
 
+def run_prediction_with_bench(input_image, model, config_file, max_percent_incorrect_values=0.1, trial=1):
+    if not input_image or not model or not config_file:
+        logger.error('Please check usage with --help option')
+        exit(1)
+    config = load_yaml(config_file)
+
+    # load the image
+    image_data = load_image(input_image)
+    raw_image = image_data
+
+    # pre process for image
+    image_data, bench_pre = _timerfunc(_pre_process, (image_data, config.PRE_PROCESSOR, config.DATA_FORMAT), trial)
+
+    # add the batch dimension
+    image_data = np.expand_dims(image_data, axis=0)
+
+    # run the model to inference
+    output, bench_inference = _timerfunc(_run, (model, image_data, config), trial)
+
+    logging.info('Output: (before post process)\n{}'.format(output))
+
+    # pre process for output
+    output, bench_post = _timerfunc(_post_process, (output, config.POST_PROCESSOR), trial)
+
+    logging.info('Output: (after post process)\n{}'.format(output))
```

Comment:
Umm, I want `total` inference time. 
`total` means pre process + network inference + post process.
But not sure everyone wants.

Reply:
ok, how about adding `total` in bench output as sum of those pre,post,inference?

Reply:
@kchygoe sound nice.

Reply:
added